gobblin-runtime/src/main/java/org/apache/gobblin/service/monitoring/JobStatusRetriever.java

```diff
@@ -105,6 +105,8 @@ protected JobStatus getJobStatus(State jobState) {
     int maxAttempts = Integer.parseInt(jobState.getProp(TimingEvent.FlowEventConstants.MAX_ATTEMPTS_FIELD, "1"));
     int currentAttempts = Integer.parseInt(jobState.getProp(TimingEvent.FlowEventConstants.CURRENT_ATTEMPTS_FIELD, "1"));
     boolean shouldRetry = Boolean.parseBoolean(jobState.getProp(TimingEvent.FlowEventConstants.SHOULD_RETRY_FIELD, "false"));
+    int progressPercentage = jobState.getPropAsInt(TimingEvent.JOB_COMPLETION_PERCENTAGE, 0);
+    long lastProgressEventTime = jobState.getPropAsLong(TimingEvent.JOB_LAST_PROGRESS_PERCENT_TIME, 0);
```

Comment:
It took me a little while to figure this out. IMO it could be named JOB_LAST_PROGRESS_EVENT_TIME